src/Bitcoin/CashAddress.h

```diff
@@ -7,22 +7,27 @@
 #pragma once
 
 #include "Address.h"
-#include "../PublicKey.h"
 #include "../Data.h"
+#include "../PublicKey.h"
+
+#include <TrustWalletCore/TWHRP.h>
 
 #include <cstdint>
 #include <string>
 
 namespace TW::Bitcoin {
 
+inline const std::string gBitcoinCashHrp{HRP_BITCOINCASH};
+inline const std::string gECashHrp{HRP_ECASH};
```

Comment:
It's better than static const variable member initialization, but require an additional include as showed above